API/go-src/main.go

```
@@ -191,7 +192,7 @@ func main() {
 		api.GET("/products/stream", verifyToken(), streamProducts)
 		api.GET("/orders/stream", verifyToken(), streamOrders)
 	}
-	r.Run(":8000")
+	r.Run(":" + config.AppPort)
```

golangci-lint (1.64.8)

Error return value of `r.Run` is not checked

(errcheck)